amf-client/shared/src/test/scala/amf/maker/DocumentMakerTest.scala

```diff
@@ -42,7 +43,7 @@ class DocumentMakerTest extends WebApiMakerTest {
 
   private def assertFixture(expected: Document, file: String, hint: Hint): Future[Assertion] = {
     Validation(platform).flatMap { v =>
-      AMFCompiler(basePath + file, platform, hint, eh = UnhandledParserErrorHandler)
+      AMFCompiler(basePath + file, platform, hint, eh = IgnoreErrorHandler)
```

Comment:
Why start ignoring errors in this test?

Reply:
This test is not for check the if the example is valid or not, just compare the parsing result against a pure, in memory, document

Reply:
So? Do you need to test invalid APIs in this suite?

Reply:
Not specifically invalid cases, but I want to remove empty endpoints, that now is being parsed and populated in the model, and not makes so mucho sense add an empty list to the fixture, that is used in different cases